index.css

```diff
@@ -46,11 +47,13 @@
   justify-content: center;
   box-sizing: content-box;
   padding: 0.5em;
+  overflow: hidden;
 }
 
 .k-block-selector-button-preview img {
   object-fit: cover;
   aspect-ratio: 16 / 9;
+  border-radius: var(--rounded-sm);
```

Comment:
`Explanation`: Users probably added preview images with a white or rather light background so far. Those look a bit lost in dark mode, which we can't fix easily. But the small border radius reduces the harshness at least a bit. 

![Screenshot 2025-05-21 at 11 05 56](https://github.com/user-attachments/assets/096fc942-872f-497f-9990-9776d6fb4ee9)